integration/simple-long/simple.ts

```diff
@@ -226,7 +226,7 @@ export const SimpleWithMap = {
     message.longLookup = {};
     if (object.longLookup !== undefined && object.longLookup !== null) {
       Object.entries(object.longLookup).forEach(([key, value]) => {
-        message.longLookup[key] = Long.fromString(value as string);
+        message.longLookup[key] = Long.fromValue(value as string);
```

Comment:
Given we're calling `fromValue` now we can/should remove the `as string`?

Reply:
Hm, maybe just `as Long | string` since those are the cases we intend to support? I think if I saw just `as string` at some point in the future, I'd forgot about this PR and think "well this should be just `Long.fromString`. :-) But the `as string | Long` I think would be a good reminder of our intent.

Reply:
Yes, good idea! I know the problem :-D I changed it and also added a link to this comment in the commit message.